src/views/About/contributors.json

```diff
@@ -38,5 +38,9 @@
   {
     "title": "Software Development Team, 2021/2022",
     "body": "Bethany Ruggerio '23, Cameron Abbot '23, Kaiwen Chu '23, Hyungyu Park '22, Silas White '23\nBennett Forkner, Evan Platzer '20, Information Systems Group"
+  },
+  {
+    "title": "Software Development Team, Summer 2023",
+    "body": "Eric Mo '24"
```

Comment:
This is unrelated to your changes. When this is added, it should include the entire team. I think you guys usually do it near the end of the program.